pkg/modules/registry/registry_test.go

```diff
@@ -9,31 +9,8 @@ import (
 
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/modules"
-	"github.com/grafana/grafana/pkg/server/backgroundsvcs"
 )
 
-func TestProvideRegistry(t *testing.T) {
```

Comment:
@wbrowne it seemed like this test was going to be too difficult to maintain as we add new dependencies to the registry. do you it's ok to rely on `TestNewRegistry`, or should we look into adding mocks for each dependency we add?

Reply:
Nah I think it's fine to kill this for now. An alternative is to just comment it out and when things are more set in stone, we can revisit and update/remove.